<launch>
    <node pkg="master_layer" name="motion_server" type="motion_server" />
    <node pkg="master_layer" name="master" type="master" output="screen" />
    <node pkg="master_layer" name="test_node" type="test_node" />
    <include file="$(find master_layer)/launch/parameters.launch" />
</launch>
